Gemfile

```diff
@@ -1,4 +1,6 @@
 source "https://rubygems.org"
 
+gem 'rails', '< 5.1.0'
```

Comment:
If we keep the default in the `gemspec` as I mentioned below I don't think this is needed in here.